### DISCUSSION TITLE
Object Simplification

### DISCUSSION THREAD

**Discussion by rickygill00** [ansys/pyaedt#5308] [General]
In Maxwell 3D is there a way to simplify a thin 3D object into a 2D sheet with a specified thickness so that it is easier to mesh?

**Comment by tizianrot** [comment]
Hallo, yes it works through Boundary Conditions. It could be either ThinLayer or Resistive Sheet. It depends on the solver you need to use and if you need to model an airgap or a conductive/resistive sheet. Please refer to the Maxwell Help:
Maxwell Help > Assigning Boundaries and Excitations for 3D Designs
I hope it helps, BR

**Reply by rickygill00** [reply]
Can this also work for windings linked to an external circuit (I.e. the windings are thin pcb traces)

**Comment by jvela018** [comment]
@rickygill00 ,

This discussion will be closed. Feel free to reopen it if further assistance is needed.

/jvela018
